src/index.html

```diff
@@ -11,7 +11,176 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+    >
+
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <label for="name">Surname:
```

Comment:
Check the code style, please
```suggestion
        <label for="name">
          Surname:
```